infra/scripts/createlocalsettings.ps1

```diff
@@ -0,0 +1,29 @@
+$ErrorActionPreference = "Stop"
+
+if (-not (Test-Path ".\app\local.settings.json")) {
+
+    $output = azd env get-values
+
+    # Parse the output to get the endpoint values
+    foreach ($line in $output) {
+        if ($line -match "AZURE_AISEARCH_ENDPOINT"){
+            $AISearchEndPoint = ($line -split "=")[1] -replace '"',''
+        }
+        if ($line -match "AZURE_OPENAI_ENDPOINT"){
+            $OpenAIEndPoint = ($line -split "=")[1] -replace '"',''
+        }
+    }
+
+    @{
+        "IsEncrypted" = "false";
+        "Values" = @{
+            "AzureWebJobsStorage" = "UseDevelopmentStorage=true";
+            "FUNCTIONS_WORKER_RUNTIME" = "dotnet-isolated";
```

Comment:
```suggestion
            "FUNCTIONS_WORKER_RUNTIME" = "python";
```

Must be set to python here in local.settings.json or your function app will not start locally